community/contributing/docs_writing_guidelines.rst

```diff
@@ -0,0 +1,345 @@
+documentation but don't add any information:
+
+.. highlight:: markdown
+
+    **Basically** a big texture [...]
+
+**Do** write short sentences in a simple, descriptive language:
+
+.. highlight:: markdown
+
+    A big texture [...]
+
+Ban these 8 words
+~~~~~~~~~~~~~~~~~
+
+**Don't** ever use these 8 banned words:
+
+1. obvious
+2. simple
+3. basic
+4. easy
+5. actual
+6. just
+7. clear
+8. however (some uses)
+
+Game creation and programming aren't simple, and nothing's easy to
+someone learning to use the API for the first time. Other words in the
+list, like ``just`` or ``actual`` won't add any info to the sentence.
+Don't use corresponding adverbs either: obviously, simply, basically,
+easily, actually, clearly.
+
+**Don't** example. The banned words lengthen the description and take
+attention away from the most important info:
+
+.. highlight:: markdown
+
+    **TextureRect**
+    Control frame that **simply** draws an assigned texture. It can stretch or not. It's a **simple** way to **just** show an image in a UI.
+
+**Do** remove them:
+
+.. highlight:: markdown
+
+    **TextureRect**
+    [Control] node that displays a texture. The texture can stretch to the node's bounding box or stay in the center. Useful to display sprites in your UIs.
+
+"Simple" never helps. Remember, for other users, anything could be
+complex or frustrate them. There's nothing like a good old *it's simple*
+to make you cringe. Here's the old brief description, the first sentence
+on the Timer node's page:
+
+.. highlight:: markdown
+
+    **Timer**
+    A **simple** Timer node.
+
+**Do** explain what the node does instead:
+
+.. highlight:: markdown
+
+    **Timer**
+    Calls a function of your choice after a certain duration.
+
+**Don't** use "basic", it is too vague:
+
+.. highlight:: markdown
+
+    **Vector3**
+    Vector class, which performs **basic** 3D vector math operations.
+
+**Do** use the brief description to offer an overview of the node:
+
+.. highlight:: markdown
+
+    **Vector3**
+    Provides essential math functions to manipulate 3D vectors: cross product, normalize, rotate, etc.
+
+Use explicit references
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Favor explicit references over implicit ones.
+
+**Don't** use words like "the former", "the latter", etc. They're not
+the most common in English, and they require you to check the reference.
+
+.. highlight:: markdown
+
+    'w' and 'h' define right and bottom margins. The **latter** two resize the texture so it fits in the defined margin.
+
+**Do** repeat words. They remove all ambiguity:
+
+.. highlight:: markdown
+
+    'w' and 'h' define right and bottom margins. **'w' and 'h'** resize the texture so it fits the margin.
+
+If you need to repeat the same variable name 3 or 4 times, you probably
+need to rephrase your description.
+
+Abuse 's to show possession
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Avoid "The milk of the cat" feels unnatural in English. Write "The cat's
+milk" instead.
```

Comment:
This sentence is awkward. (Nitpick: cat milk?)

Avoid saying "The milk of the cow", it feels unnatural in English. Instead, use "The cow's milk".

Reply:
Cats drink milk! Ours does, at least ;). Sure, gonna fix it

Reply:
I just got a mental image of someone *milking* a cat, not feeding it! :D